src/go/rdctl/pkg/factoryreset/delete_data.go

```diff
@@ -30,11 +30,16 @@ import (
 	"syscall"
 
 	dockerconfig "github.com/docker/docker/cli/config"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/autostart"
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
 	"github.com/sirupsen/logrus"
 )
 
 func DeleteData(removeKubernetesCache bool) error {
+	if err := autostart.EnsureAutostart(false); err != nil {
```

Comment:
This should be called in src/go/rdctl/pkg/factoryreset/factory_reset_windows.go instead because it's platform-specific.

Reply:
Similar to the [above case](https://github.com/rancher-sandbox/rancher-desktop/pull/3798#discussion_r1097755711), this will need to be called on all three platforms, and this function will exist for all three platforms. It's just that at the moment, this function returns an error for Linux and macOS. I will make it a no-op for these platforms.